db/migrations/0002.add_col_published_date.py

```diff
@@ -0,0 +1,8 @@
+#
+# file: migrations/0002.add_col_published_date.py
+#
+from yoyo import step
+
+__depends__ = {'0001.init_inventory_db'}
+
+step("ALTER TABLE course ADD column published_date VARCHAR(25) AFTER created_at")
```

Comment:
Thinks were simple with migration, I am Q though why is this VARCHAR instead of datetime, I just followed what was for `created_Date`.

Reply:
Yeah, it should probably be `TIMESTAMP` or one of the other time-related data types. I encountered problems when trying to insert the data into the database, so I just forged ahead with VARCHAR for the moment. If you can use something other than VARCHAR, please do. I can work on getting the other one changed back. Thanks for reminding me.

Reply:
It's all boils down to how tableau users wants the data to be like. it will make sense both date to be same data type otherwise on tableau end they might need to do some tableau calculation or tweaks to adjust to same data type.

Reply:
Agreed, it should be consistent. You can keep it the same data type for now, and I can work on improving both later.